pkg/cfn/manager/api.go

```diff
@@ -348,33 +348,31 @@ func (c *StackCollection) ListStacksMatching(nameRegex string, statusFilters ...
 		StackStatusFilter: defaultStackStatusFilter(),
 	}
 	if len(statusFilters) > 0 {
-		input.StackStatusFilter = aws.StringSlice(statusFilters)
+		input.StackStatusFilter = statusFilters
 	}
 	stacks := []*Stack{}
 
-	pager := func(p *cloudformation.ListStacksOutput, _ bool) bool {
-		for _, s := range p.StackSummaries {
-			if re.MatchString(*s.StackName) {
-				stack, subErr = c.DescribeStack(&Stack{StackName: s.StackName, StackId: s.StackId})
-				if subErr != nil {
-					return false
-				}
-				stacks = append(stacks, stack)
-			}
-		}
-		return true
-	}
-	if err := c.cloudformationAPI.ListStacksPages(input, pager); err != nil {
+	allStacks, err := c.cloudformationAPI.ListStacks(ctx, input)
```

Comment:
Hm.... I love that we are removing the Pages thing, however, this also means that this will not paginate. That might be a problem in case of many stacks, not? The Pages functions did pagination for us.

Reply:
The plain `ListStacks` does not paginate I'm afraid.